vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -202,6 +206,19 @@ void ReactInstanceWin::Initialize() noexcept {
               std::move(m_appearanceListener),
               m_legacyReactInstance);
 
+          auto nmp = std::make_shared<winrt::Microsoft::ReactNative::NativeModulesProvider>();
+          nmp->AddModuleProvider(
+              L"Clipboard",
+              winrt::Microsoft::ReactNative::MakeTurboModuleProvider<
+                  ::Microsoft::ReactNative::Clipboard,
+                  ::Microsoft::ReactNativeSpecs::ClipboardSpec>());
+
+          auto modules = nmp->GetModules(m_reactContext, m_batchingUIThread);
+          for (auto mod : modules) {
+            auto [name, provider, queue] = mod;
+            cxxModules.emplace_back(name, provider, queue);
+          }
```

Comment:
It seems that we do too many copies here.
Consider to have something as the following:
```
cxxModules.insert(cxxModules.end(),std::make_move_iterator(modules.begin()),
         std::make_move_iterator(modules.end()));
```